### DISCUSSION TITLE
Stuck in project pending review.

### DISCUSSION THREAD

**Discussion by MrBurnsDev** [WalletConnect/walletconnect-monorepo#4645] [Developer Support]
My project has been stuck in pending review for 2 weeks and I'm unsure if this is normal. The project ID is d771164ba8107e6f84a342ed10915afa. 

I believe that all of guidelines have been met. 

Thanks in advance!

**Comment by glitch-txs** [comment]
Hello, I'm seeing your project as approved

**Reply by MrBurnsDev** [reply]
I still see this on my end. ![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/59661382/0b53593e-d074-4459-95e2-e3e93be69016)

**Reply by MrBurnsDev** [reply]
We were recently getting JSON-RPC error.

**Reply by glitch-txs** [reply]
are you sure you shared the correct project ID?
